root/static/scripts/tests/react-macros.js

```diff
@@ -0,0 +1,155 @@
+// This file is part of MusicBrainz, the open internet music database.
+// Copyright (C) 2016 MetaBrainz Foundation
+// Licensed under the GPL version 2, or (at your option) any later version:
+// http://www.gnu.org/licenses/gpl-2.0.txt
+
+'use strict';
+
+require('babel-core/register');
+
+const document = require('jsdom').jsdom();
+const React = require('react');
+const ReactDOMServer  = require('react-dom/server');
+
+const DescriptiveLink = require('../common/components/DescriptiveLink');
```

Comment:
These imports are there 'cause on line 117 we `eval` code that the perl sent to us in t/lib/t/TemplateMacros.pm, which references these components.